src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts

```diff
@@ -1137,6 +1137,8 @@ export class ExplorePageComponent implements OnInit, OnDestroy, AfterViewInit {
             if(pillData.name === 'observation'){
                 this.router.navigate(['observation']);
             }
+        }else{
+            window.location.href = '/observation';
```

Comment:
- if  pillData.name === 'observation' then you are redirecting to observation
- if non logged in user also; you are redirecting to observation.
Please check the condition 
CC: @rajeshkumaravel